dspinner/pin_test.go

```diff
@@ -664,14 +664,17 @@ func TestLoadDirty(t *testing.T) {
 	cidRIndex.DeleteKey(ctx, cidAKey)
 	cidRIndex.Add(ctx, cidBKey, "not-a-pin-id")
 
+	// Force dirty, since Pin syncs automatically
+	p.(*pinner).setDirty(ctx, true)
+
 	// Verify dirty
-	data, err := dstore.Get(dirtyKey)
-	if err != nil {
-		t.Fatalf("could not read dirty flag: %v", err)
-	}
-	if data[0] != 1 {
-		t.Fatal("dirty flag not set")
-	}
+	// data, err := dstore.Get(dirtyKey)
+	// if err != nil {
+	// 	t.Fatalf("could not read dirty flag: %v", err)
+	// }
+	// if data[0] != 1 {
+	// 	t.Fatal("dirty flag not set")
+	// }
```

Comment:
Remove? Although there shouldn't be anything wrong with these checks if we want to uncomment them

Reply:
Uncommented in https://github.com/ipfs/go-ipfs-pinner/pull/13/commits/a24f5872bea10d592361febcd6bb3bf0ea5743e0.